target/start-mailserver.sh

```diff
@@ -586,6 +587,12 @@ function _setup_dovecot() {
 	sed -i -e 's/#ssl = yes/ssl = required/g' /etc/dovecot/conf.d/10-ssl.conf
 	sed -i 's/^postmaster_address = .*$/postmaster_address = '$POSTMASTER_ADDRESS'/g' /etc/dovecot/conf.d/15-lda.conf
 
+	# Set mail_max_userip_connections
+	if [[ ! -z ${DEFAULT_VARS["DOVECOT_MAX_USERIP_CONNECTIONS"]} ]]; then
```

Comment:
Default value is 10. So this test is always true.
Better would be: `if [ ${DEFAULT_VARS["DOVECOT_MAX_USERIP_CONNECTIONS"]} != "10" ]; then`

Reply:
Good catch!  Thanks!

Reply:
Do you think it would be better to make your change or remove the line from the DEFAULT_VARS section altogether?

Reply:
@erik-wramner is right, I don't think we need that configurable via a variable. I just wanted you to know that there is a logical error in your code.
Regarding your last question, I would stay with the default variable defined. That way, it's easier to keep track, which variables are used in the script.